textdb/textdb-dataflow/pom.xml

```diff
@@ -59,13 +59,16 @@
             <version>3.6.0</version>
             <classifier>models</classifier>
         </dependency>
-
-
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-all</artifactId>
             <version>1.9.5</version>
         </dependency>
+        <dependency>
+            <groupId>info.debatty</groupId>
+            <artifactId>java-string-similarity</artifactId>
```

Comment:
introduce the `java-string-similarity` dependency for various string similarity functions.